libs/utils/src/decorators/is-schema-name.decorator.ts

```diff
@@ -0,0 +1,22 @@
+import { SCHEMA_NAME_TO_ID } from '#types/constants/schemas';
+import { registerDecorator, ValidationOptions, ValidationArguments } from 'class-validator';
+
+export function IsSchemaName(validationOptions?: ValidationOptions) {
+  // eslint-disable-next-line func-names
+  return function (object: Object, propertyName: string) {
+    registerDecorator({
+      name: 'IsSchemaName',
+      target: object.constructor,
+      propertyName,
+      options: validationOptions,
+      validator: {
+        validate(value: any) {
+          return SCHEMA_NAME_TO_ID.has(value);
```

Comment:
nit: we could sanitize the value like trim it or make it all lower case. This should probably happen during transformation if it's easy to do.

Reply:
I'm ok for now with this being specific, but we should look at that for several places.